README.md

```diff
@@ -245,9 +245,9 @@ Must match the value specified in the server.</td></tr><tr><th>Required:</th><td
 </tbody></table>
         </details><details><summary>ProfileTransaction (<code>object</code>)</summary>
             <table><tbody><tr><th>Type:</th><td><code>object</code></td><tr><th>Properties</th><td><details><summary>duration (<code>string</code>)</summary>
-        <table><tbody><tr><th>Name:</th><td>duration</td></tr><tr><th>Description:</th><td>How long the items in the transaction run.</td></tr><tr><th>Required:</th><td>No</td></tr><tr><th>Type:</th><td><code>string</code></td></tbody></table>
+        <table><tbody><tr width=600><th>Name:</th><td>duration</td></tr><tr><th>Description:</th><td width="500">How long the items in the transaction run.</td></tr><tr width=600><th>Required:</th><td>No</td></tr><tr><th>Type:</th><td><code>string</code></td></tbody></table>
         </details><details><summary>flowops (<code>list[<code>one of[string]</code>]</code>)</summary>
-        <table><tbody><tr><th>Name:</th><td>flowops</td></tr><tr><th>Description:</th><td>The flowop name for uperf.
+        <table><tbody><tr width=600><th>Name:</th><td>flowops</td></tr><tr><th>Description:</th><td width="500">The flowop name for uperf.
 - connect: Specifies that a connection needs to be opened.
 - accept: Specifies that a connection needs to be accepted from the remote
 - disconnect: Disconnects from the existing connection.
```

Comment:
Two comments.

First, why is `"500"` in quotes when `600` is not?

Second, the "bullets" and line breaks in this description are lost when it is rendered, and the result is hard to read.  (I'm not sure what to suggest to fix it, but what we're getting now is not very good.)

Reply:
I overlooked changes I needed to revert. The quote inconsistency is just me being indecisive, but that's fixed when the `<tr>` widths are taken away, so I'll fix that.

The "bullets" you're referring to have nothing to do with the docsgen change and are due to a bad definition in the schema that doesn't take into account that the multi-line string will be collapsed into a single line: https://github.com/arcalot/arcaflow-plugin-uperf/blob/57e6406a7fd4948c5763c9df9c5c98c5deeb91ce/arcaflow_plugin_uperf/uperf_schema.py#L161-L175

Reply:
Tracked in [ARCA-418](https://issues.redhat.com/browse/ARCA-418).